sound/soc/sof/intel/hda-loader.c

```diff
@@ -78,26 +78,14 @@ static int cl_stream_prepare(struct snd_sof_dev *sdev, unsigned int format,
  * reset/stall and then turn it off
  */
 static int cl_dsp_init(struct snd_sof_dev *sdev, const void *fwdata,
-		       u32 fwsize)
+		       u32 fwsize, int tag)
```

Comment:
Can you add comments for the  parameter "tag", like "tag of the stream to download the firmware"?